### DISCUSSION TITLE
Pregunta sobre punteros

### DISCUSSION THREAD

**Discussion by Clirussa** [INGCOM-UNRN-P1/cursada-2023#192] [Preguntas y respuestas] [answered]
Hola, tengo una pregunta para usar `NULL` ¿Se debe definir la librería `<stddef.h>` o con `<stdio.h>` debería poder usarse? ¿Cómo me doy cuenta?

Gracias

**Comment by martinvilu** [comment, marked as answer]
Es muy raro usar punteros completamente solos, la mejor opción es incluir `stdlib` que de paso trae `malloc`.

De todas formas, con cualquiera de las "`std`" alcanza.

**Reply by Clirussa** [reply]
Gracias
